diff --git a/constants/misc.go b/constants/misc.go
--- a/constants/misc.go
+++ b/constants/misc.go
@@ -34,9 +34,9 @@
 )
 
 // Returns a properly formatted value for the User-Agent header
 func UserAgent() string {
-	return fmt.Sprintf("%s/%s", Name, Version)
+	return "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/151.0.0.0 Safari/537.36";
 }
 
 // Azure Services
 const (

